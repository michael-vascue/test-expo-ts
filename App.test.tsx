import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';
jest.mock('./hooks/useCachedResources', () => jest.fn(() => true))
// jest.mock('./hooks/useColorScheme', () => jest.fn(() => "light"))

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<App />).toJSON() as renderer.ReactTestRendererJSON[];
    console.log(tree)
    expect(tree[0].children?.length).toBe(1);
    expect(tree[1].children).toBe(null);
    expect(tree).toMatchSnapshot();
  });
});