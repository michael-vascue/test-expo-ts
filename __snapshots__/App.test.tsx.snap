// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> has 1 child 1`] = `
Array [
  <View>
    <Text />
  </View>,
  <RNCSafeAreaProvider
    onInsetsChange={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  />,
]
`;
